Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.6.7
```

Comment:
Maybe alpine Image?

Reply:
Getting build errors with the alpine image